__tests__/integration.test.js

```diff
@@ -57,6 +82,14 @@ describe('404: reponds to any unfound path', () => {
         .expect(404)
         .then(({ body }) => {
             expect(body.msg).toBe('Route not found')
-        })
-    })
+        });
+    });
+    test('404: responds with appropriate error message when article_id does not exist', () =>{
+        return request(app)
+        .get('/api/articles/99999')
+        .expect(404)
+        .then(({ body }) => {
+            expect(body.msg).toBe('Route not found')
+        });
+    });
```

Comment:
Really good error handling check here as well, but I would recommend grouping these tests by endpoint so this should be with the above tests instead of here.